content/glossary.md

```diff
@@ -819,37 +821,39 @@ External links: [Well-being](https://plato.stanford.edu/entries/well-being/), Th
+[^31]: Smart, R. N. (1958). [Negative Utilitarianism](https://doi.org/10.1093/mind/LXVII.268.542). _Mind_. 67(268): 542–43.
+[^32]: Bentham, J. (1789). [An Introduction to the Principles of Morals and Legislation](https://www.earlymoderntexts.com/assets/pdfs/bentham1780.pdf). Bennet, J. (ed.)., p. 7
+[^33]: More precisely: the more that an act would promote the sum total of well-being, the more moral reason one has to perform that act.
+[^34]: Norcross, A. (2020). _Morality by Degrees: Reasons Without Demands_. Oxford University Press.
+[^35]:
     Jeremy Bentham rejected single-level utilitarianism, writing that “it is not to be expected that this process [of calculating expected consequences] should be strictly pursued previously to every moral judgment.” Bentham, J. (1789). [An Introduction to the Principles of Morals and Legislation](https://www.earlymoderntexts.com/assets/pdfs/bentham1780.pdf). Bennet, J. (ed.)., p. 23
+
     Henry Sidgwick concurs, writing that “the end that gives the criterion of rightness needn’t always be the end that we consciously aim at; and if experience shows that general happiness will be better achieved if men frequently act from motives other than pure universal philanthropy, those other motives are preferable on utilitarian principles”. Sidgwick, H. (1874). [The Methods of Ethics](https://www.earlymoderntexts.com/assets/pdfs/sidgwick1874.pdf). Bennet, J. (ed.)., p. 201
 
-[^35]: Whether or not we should have the child, however, depends also on whether this improves the total well-being more than improving the lives of existing people would, and on issues regarding resource constraints and overpopulation.
-[^36]: This definition applies to a fixed-population setting, where one’s actions do not affect the number or identity of people. There are utilitarian theories that differ in how they deal with variable-population settings. This is a technical issue, relevant to the discussion of [population ethics](https://en.wikipedia.org/wiki/Population_ethics).
-[^37]: Nozick, R. (1974). _[Anarchy, State, and Utopia](https://en.wikipedia.org/wiki/Anarchy,_State,_and_Utopia)_. Basic Books.
+[^36]: Whether or not we should have the child, however, depends also on whether this improves the total well-being more than improving the lives of existing people would, and on issues regarding resource constraints and overpopulation.
+[^37]: This definition applies to a fixed-population setting, where one’s actions do not affect the number or identity of people. There are utilitarian theories that differ in how they deal with variable-population settings. This is a technical issue, relevant to the discussion of [population ethics](https://en.wikipedia.org/wiki/Population_ethics).
```

Comment:
Should we link to both, our internal _Population Ethics_ page and external link?

Reply:
Just our internal chapter will do -- thanks for catching the outdated link!